crates/cli/src/subcommands/login.rs

```diff
@@ -120,43 +120,76 @@ async fn spacetimedb_token_cached(
 }
 
 async fn web_login_cached(config: &mut Config, host: &str, clear_cache: bool) -> anyhow::Result<String> {
-    let session_id = config.web_session_id().filter(|_| !clear_cache);
-    if let Some(session_id) = session_id {
+    let session_token = config.web_session_token().filter(|_| !clear_cache);
+    if let Some(session_token) = session_token {
         // Currently, these session IDs do not expire. At some point in the future, we may also need to check this session ID for validity.
-        Ok(session_id.clone())
+        Ok(session_token.clone())
     } else {
-        let session_id = web_login(host).await?;
-        config.set_web_session_id(session_id.clone());
+        let session_token = web_login(host).await?;
+        config.set_web_session_token(session_token.clone());
         config.save();
-        Ok(session_id)
+        Ok(session_token)
     }
 }
 
 #[derive(Deserialize)]
-struct WebLoginTokenResponse {
+struct WebLoginTokenData {
     token: String,
 }
 
+#[derive(Deserialize)]
+struct WebLoginTokenResponse {
+    success: bool,
+    data: WebLoginTokenData,
+}
```

Comment:
nit: is it really necessary/useful to have this wrapper type if it just has a single member?

Reply:
I have standardized all the api routes with this structure, so consumer can check with a simple .success check whether it worked or not. Previously they were all being done either by returning json success result, or a text error, which would cause deserializing errors anywhere. This solid structure makes sure it can be avoided

Reply:
Is this a general standard for all of our API routes, or just the auth-related ones? I'm not seeing this same pattern in other areas of our repo.

Reply:
This is for anything on the website

Reply:
I notice that `/login/request-token` endpoint doesn't return the `error` member - is that on purpose?

Reply:
There isn’t anything on it that could give an error. Maybe I can add something but for now it’s not that worth it

Reply:
Ohhh. I mean I dont have any opinion, I just keep these things an object so later if we add more properties we don't break stuff. Prolly doesn't apply here, its just part of my personal philosophy